pkg/cloud/cloud.go

```diff
@@ -498,6 +508,7 @@ func (c *cloud) GetDiskByID(ctx context.Context, volumeID string) (*Disk, error)
 		VolumeID:         aws.StringValue(volume.VolumeId),
 		CapacityGiB:      aws.Int64Value(volume.Size),
 		AvailabilityZone: aws.StringValue(volume.AvailabilityZone),
+		OutpostArn:       aws.StringValue(volume.OutpostArn),
```

Comment:
does GetDiskByName also need this change? IT looks like GetDiskByName can get called in CreateVolume

Reply:
could verify this with a test case where GetDiskByName does return something instead of errnotfound

Reply:
Great catch! If the volume exists, we would return the existing disk without the outpost arn. I've updated `GetDiskByName` and added a test case. Should be good now.